### ISSUE TITLE
How to expose webpack-dev-server externally?

### ISSUE THREAD

**Issue by OriginalEXE** [lando/lando#856]
Question
==========

**Tell us about your setup**

v3.0.0-beta.39 on Ubuntu 16.04 LTS

**Tell us about your `.lando.yml`**

```yml
name: rough-hands-barbershop
recipe: wordpress
config:
  webroot: web
events:
  post-start:
    - appserver: cd $LANDO_MOUNT/web/wp-content/themes/as-rough-hands-barbershop && composer install
    - nodeserver: cd $LANDO_MOUNT && chmod +x ./build.sh && chmod +x ./dev.sh
services:
  nodeserver:
    type: node:8.4
    overrides:
      services:
        ports:
          - "9000:9000"
    run:
      - "cd $LANDO_MOUNT/web/wp-content/themes/as-rough-hands-barbershop && npm install"
tooling:
  devstart:
    service: nodeserver
    cmd: ./dev.sh
  buildstart:
    service: nodeserver
    cmd: ./build.sh
proxy:
  nodeserver:
    - assets.lndo.site:9000
```

**Tell us about the issue you got**
Hello,

I am trying to use Lando for developing a WordPress theme. I have used a WordPress recipe successfully and got it running, that works perfectly. Now, I am trying to use webpack, or more specifically webpack-dev-server, to serve my assets while in development. To achieve that, I have created a new service called nodeserver, where I then run webpack-dev-server. webpack-dev-server exposes assets on `localhost:9000`, however, I can't reach that port externally. I have pasted my config above which contains my latest attempt at achieving this.

So basically what I'm trying to do is expose :9000 from my nodeserver service externally.

Thanks a lot!

**Comment by OriginalEXE** [comment]
I have solved this.

The problem was that I was binding the wrong port, here is the correct version:

```yml
name: rough-hands-barbershop
recipe: wordpress
config:
  webroot: web
events:
  post-start:
    - appserver: cd $LANDO_MOUNT/web/wp-content/themes/as-rough-hands-barbershop && composer install
    - nodeserver: cd $LANDO_MOUNT && chmod +x ./build.sh && chmod +x ./dev.sh
services:
  nodeserver:
    type: node:8.4
    overrides:
      services:
        ports:
          - "8080:8080"
    run:
      - "cd $LANDO_MOUNT/web/wp-content/themes/as-rough-hands-barbershop && npm install"
tooling:
  devstart:
    service: nodeserver
    cmd: ./dev.sh
  buildstart:
    service: nodeserver
    cmd: ./build.sh
```

Also, in webpack dev server config add this:
```javascript
host: '0.0.0.0',
disableHostCheck: true,
```
